README.md

```diff
@@ -14,6 +14,20 @@ of the repository.
 Each app has a requirements.txt, install the dependecies in a virtual
 environment.
 
+## Downloading and running a single app
+
+If you don't want to clone the entire repo, you can also download an individual app as a zip file (see the [releases](https://github.com/plotly/dash-sample-apps/releases)), decompress and run them. In the example below, replace `<name>` by the name of the app:
+
+```bash
+wget https://github.com/plotly/dash-sample-apps/releases/latest/download/<name>.zip
```

Comment:
Do we really want to document `wget` as the main entry point? I'd think a friendlier route would be from the website like:

---

Visit the [releases](https://github.com/plotly/dash-sample-apps/releases) page and download and unzip the app you want. Then `cd` into the app directory and install its dependencies in a virtual environment like:
```bash
python -m venv venv
source venv/bin/activate  # Windows: \venv\scripts\activate
pip install -r requirements.txt
```
Then run the app:
```bash
python app.py
```

---

separate note: wouldn't we want to suggest the same pattern above when cloning the whole repo? ie `cd` into the app dir, set up the virtual env in that dir, and run the app from there? The way we describe it now with running things from the repo root makes it unclear how to handle app-specific venvs.